lib/isolator.go

```diff
@@ -64,12 +65,24 @@ func (a *ACBuild) AddIsolator(name string, value []byte) (err error) {
 		if s.App == nil {
 			s.App = newManifestApp()
 		}
+		_, ok := types.LinuxIsolatorNames[*acid]
```

Comment:
Right, the linux isolators are not there, the same reason as resource isolators are not in `LinuxIsolatorNames`.

```
$ git checkout origin/pr/240
$ ./build
Building acbuild...
$ ./bin/acbuild begin
$ echo '{ "set": ["CAP_IPC_LOCK"] }' | ./bin/acbuild isolator add "os/linux/capabilities-retain-set" -
$ echo '{ "limit": "20m" }' | ./bin/acbuild isolator add "resource/cpu" -
isolator add: unknown isolator name: resource/cpu
```